amida-tech/greyscale
# Bug/inba 757 po better notifications

- Gives better notification Text when a users task is activated
- Makes sure to check and honor a users `notificationLevel` 

TO TEST:
- Create a project, assign task
- Make sure that the email the user gets makes sense
- Change the users notification level (Directly from the DB or the front end)
- Carry out an action and verify the users notification based on the notificationLevel you set
